### PR TITLE
E key now works for the machete pouch

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

The machete pouch can be drawn like a sidearm pouch.

remake of #115 

## Why It's Good For The Game

storage items should be consistent

## Changelog

:cl: Ghostsheet
balance: you can now take machetes out of the machete pouch via hotkey
/:cl:

### CODE REVIEW

**Comment by Bram-KEP** [comment]
God bless 
Finally

**Comment by Sargeantmuffinman** [comment]
huzzzaaaaah

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 1]
```diff
@@ -102,7 +102,7 @@
 		list("Large Pistol Magazine Pouch", round(scale * 5), /obj/item/storage/pouch/magazine/pistol/large, VENDOR_ITEM_REGULAR),
 		list("Magazine Pouch", round(scale * 5), /obj/item/storage/pouch/magazine, VENDOR_ITEM_REGULAR),
 		list("Shotgun Shell Pouch", round(scale * 5), /obj/item/storage/pouch/shotgun, VENDOR_ITEM_REGULAR),
-		list("Machete Pouch (Full)", round(scale * 4), /obj/item/storage/large_holster/macheteB/full, VENDOR_ITEM_REGULAR),
+		list("Machete Pouch (Full)", round(scale * 4), , VENDOR_ITEM_REGULAR),
```
Comment (JeserTheFox) [inline code/game/machinery/vending/vendor_types/requisitions.dm:105]:
```suggestion
		list("Machete Pouch (Full)", round(scale * 4), /obj/item/storage/pouch/machete/full, VENDOR_ITEM_REGULAR),
```

**Comment by blackdragonTOW** [comment]
Finally, my 8 machete build is viable.

**Review by morrowwolf — APPROVED** [review]
LGTM

**Pull request merged**